### PR TITLE
New Extensions

### PR DESCRIPTION
**Pull request by ebrarbilgili**
-EdgeInsets.zero added as paddingZero,
-Network Images added on StringExtension as NetworkImageExtension, 
-LocalizationExtension added on StringExtension, 
-warnings cleaned

### CODE REVIEW

File: lib/src/string_extension.dart [hunk 1]
```diff
@@ -96,3 +105,16 @@ extension PackageInfoExtension on String {
     }
   }
 }
+
+extension LocalizationExtension on String {
+  String get locale => this.tr();
```
Comment (VB10) [inline lib/src/string_extension.dart:110]: maybe it's have problem as user couldnt use easy localization and it will be not work, please back this code

**Review by VB10 — CHANGES_REQUESTED** [review]

**Comment by ebrarbilgili** [comment]
localization is removed and it is committed

File: pubspec.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ dependencies:
   flutter:
     sdk: flutter
   device_info: ^1.0.0
+  easy_localization: ^3.0.0
```
Comment (VB10) [inline pubspec.yaml]: may you forget this

**Review by VB10 — CHANGES_REQUESTED** [review]

File: pubspec.yaml (hunk 2 above)
Comment (ebrarbilgili) [reply pubspec.yaml]: it should be removed in last commit

**Pull request merged**